tests/datasets_tests/cub_tests/test_cub_label_dataset.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+
+from chainer import testing
+from chainer.testing import attr
+
+from chainercv.datasets import cub_label_names
+from chainercv.datasets import CUBLabelDataset
+from chainercv.utils import assert_is_classification_dataset
+
+
+@testing.parameterize(
+    {'crop_bbox': True},
+    {'crop_bbox': False}
+)
+class TestCUBLabelDataset(unittest.TestCase):
+
+    def setUp(self):
+        self.dataset = CUBLabelDataset(crop_bbox=self.crop_bbox)
+
+    @attr.slow
+    def test_cub_label_dataset(self):
+        assert_is_classification_dataset(
+            self.dataset, len(cub_label_names), n_example=10)
+
```

Comment:
Returning `bbox` looks good to me. As you said, it will be more flexible. Perhaps, adding `transforms.image.crop_with_bbox (or simply crop)` will be helpful. With this function, we can write cropped dataset easily, 
`TransformDataset(CUBLabelDataset(return_bbox), lambda in_data: crop_with_bbox(in_data[0], in_data[2]), in_data[1])`.